modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1309,6 +1309,101 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the user is able to add the Formula Field when managing Custom Validation"
+	@priority = 4
+	test CanManageFormulaFieldWithCustomValidation {
+		property custom.properties = "feature.flag.LPS-164948=true";
+
+		task ("Given an Object with Formula Field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 176853",
+				objectName = "CustomObject176853",
+				pluralLabelName = "Custom Objects 176853");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 176853");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Formula Field Plus",
+				fieldType = "Formula",
+				formulaFieldOption = "Integer");
+
+			for (var labelName : list "A,B") {
+				ObjectAdmin.addObjectFieldViaUI(
```

Comment:
This can be accomplished using the API, for example [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomValidation.testcase#L526)